helpers/helpersTest.js

```diff
@@ -0,0 +1,46 @@
+const Helper = require('@codeceptjs/helper');
+
+class HelpersTest extends Helper {
+
+  async openStoreFromHelper() {
+    await this.helpers['Playwright'].amOnPage('http://automationpractice.com/index.php');
```

Comment:
generic functions, like `login()`, `openStore()` is better to keep in `steps_file.js`